src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.File.NotifyFilter.cs

```diff
@@ -91,9 +91,6 @@ public void FileSystemWatcher_File_NotifyFilter_DirectoryName(NotifyFilters filt
 
                 if (filter == NotifyFilters.DirectoryName)
                     expected |= WatcherChangeTypes.Renamed;
-                else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) && (filter == NotifyFilters.FileName))
-                    expected |= WatcherChangeTypes.Renamed;
```

Comment:
It is no longer needed because this test performs a `Directory.Move` which will no longer trigger an event when only `NotifyFilters.FileName` is set.